.github/workflows/build.yml

```diff
@@ -116,6 +116,8 @@ jobs:
       with:
         submodules: true
     - run: sudo apt-get install ninja-build
+    - name: workaround for ASLR+ASAN compatibility
```

Comment:
Can you add a link to https://github.com/actions/runner/issues/3207 here in comment ?